tool/tsh/common/mfa.go

```diff
@@ -350,6 +351,16 @@ func (c *mfaAddCommand) addDeviceRPC(ctx context.Context, tc *client.TeleportCli
 		if authChallenge == nil {
 			return trace.BadParameter("server bug: server sent %T when client expected AddMFADeviceResponse_ExistingMFAChallenge", resp.Response)
 		}
+
+		// Tweak Windows platform messages so it's clear we whether we are prompting
+		// for the *registered* or *new* device.
+		// We do it here, preemptively, because it's the simpler solution (instead
+		// of finding out whether it is a Windows prompt or not).
+		const registeredMsg = "Using platform authentication for *registered* device, follow the OS dialogs"
```

Comment:
```suggestion
		const registeredMsg = "Using platform authentication for *existing* device, follow the OS dialogs"
```
I think this is a tiny bit easier to understand.

Reply:
We do use that language in other places, and have been using it for a long time, so I think I'd rather keep it?